Aleksey Filatov/func6.js

```diff
@@ -0,0 +1,37 @@
+//Largest function ES5 sorting
+
+function largest() {
+   var array = Array.from(arguments);
```

Comment:
@AlexFil1983 According to the main tasks es6 is prohibited, so please don't use Array.from ;)